projects/reservoir/fv3/save_ranks.py

```diff
@@ -13,6 +13,11 @@
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
+# Shift hybrid data by this number of timesteps.
+# -1 is common use case for saving the next timestep's ML prediction at
+# the same sample index as reservoir input.
+TIME_SHIFT_HYBRID_DATA = -1
```

Comment:
Is there any benefit to adding this as a configuration option with a default as opposed to a hard-code?

Reply:
In practice we'll always have the t+1 hybrid input available at run time, so I'm not sure if we'd want to train models with different time shifts. It'd be a good reminder of what the code is doing though to have this in the args with a default of -1.